packages/react-components/src/components/tabs/VerticalTab.tsx

```diff
@@ -0,0 +1,51 @@
+import React,  {useContext, useCallback, useState} from 'react';
+import {FormContext} from '@aemforms/af-react-renderer';
+import {getRenderer, useFormIntl} from '@aemforms/af-react-renderer';
+import {ContainerJson, State, CUSTOM_PROPS_KEY, TRANSLATION_ID, getOrElse} from '@aemforms/af-core';
+
+const VerticalTab = (props: State<ContainerJson>) => {
+   const {mappings} = useContext(FormContext);
+   const [activetabIndex, setActiveTabIndex] = useState(0);
+   const { items, label, id, visible, enabled } = props;
+   const {v: visibleItems} =
+   items.reduce(({v} : any, item) => {
+       const isVisible = item.visible === true;
+       return {
+           v: isVisible ? v.concat([item]) : v
+       };
+   }, {v: []});
+
+const localizeTabLabel = useCallback((item: State<ContainerJson>) => {
+   let defaultMessage = item?.label?.value;
+   const id = getOrElse(item, [CUSTOM_PROPS_KEY, TRANSLATION_ID, 'label.value']);
+   if (id) {
+       const i18n = useFormIntl();
+       defaultMessage = i18n.formatMessage({id, defaultMessage});
+   }
+   return defaultMessage;
+}, [useFormIntl]);
+
+const handleClick = (index: number) => {
+   setActiveTabIndex(index);
+}
+
+const Comp = getRenderer(visibleItems[activetabIndex], mappings);
```

Comment:
If fieldType/:type is not available in the mappings then the getRenderer method return null. please add null check